ffmpeg/cppbuild.sh

```diff
@@ -1163,7 +1163,7 @@ EOF
         make -j $MAKEJ
         make install
         cd ../libwebp-$WEBP_VERSION
-        $CMAKE -DCMAKE_C_FLAGS="$CFLAGS -fPIC" -DCMAKE_C_COMPILER=aarch64-linux-gnu-gcc -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH $WEBP_CONFIG .
+        CFLAGS="-I$INSTALL_PATH/include/" CXXFLAGS="-I$INSTALL_PATH/include/" LDFLAGS="-L$INSTALL_PATH/lib/" $CMAKE -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH $WEBP_CONFIG -DCMAKE_SYSTEM_NAME=Linux -DCMAKE_SYSTEM_VERSION=1 -DCMAKE_SYSTEM_PROCESSOR=armv8 -DCMAKE_CXX_FLAGS="$CXXFLAGS -fPIC" -DCMAKE_C_FLAGS="$CFLAGS -fPIC" -DCMAKE_C_COMPILER=aarch64-linux-gnu-gcc -DCMAKE_CXX_COMPILER=aarch64-linux-gnu-g++ .
```

Comment:
Why the `CXX` bits (in general, where this is just one specific case)? I don't think there is any C++ in libwebp.

Reply:
Why not? These flags should eventually be all cleaned up and put into one place. No one just ever got to it, but for now, there's no point in having inconsistent flags all over the place.